### PR TITLE
Issue-45:Opentsdb component healthmetric information is not being displayed in console homepage.

### PR DESCRIPTION
**Pull request by Deepa-Martin-ML**
**Problem Statement:**
ISSUE 45: Opentsdb component healthmetric information is not being displayed in console homepage.

**Analysis:**
There are no proper UI representation of information related to OpenTSDB. Presently, OpenTsdb has one URL on console homepage which redirects to another page. User cannot see the health status of opentsdb. 

**Change:**
Added UI represntation of opentsdb. It is a block like structure on homepage which has a color code which depends on health status of openTsdb. That block has three icon which are info, help and settings respectively.

### CODE REVIEW

File: console-frontend/help/topics.json [hunk 1]
```diff
@@ -69,6 +69,11 @@
         "Impala is only available with the Cloudera distribution of Hadoop"],
       "link": "http://impala.io"
     },
+     "opentsdb.health": {
+      "body": ["OpenTSDB is a scalable time series database that lets you store and serve massive amounts of time series data,without losing granularity.",
```
Comment (jeclarke) [inline console-frontend/help/topics.json]: comma needed "data,without"

File: console-frontend/js/controllers/MetricsListCtrl.js [hunk 2]
```diff
@@ -305,15 +305,19 @@ angular.module('appControllers').controller('MetricListCtrl', ['$scope', 'Metric
...
         if (source === "kafka") {
           resolutionUrl = $scope.dm_endpoints.kafka_manager;
         } else if (source === "deployment-manager") {
           resolutionUrl = ConfigService.userInterfaceIndex["PNDA logserver"];
-        } else if (source === "opentsdb" || source === "grafana") {
-          // for opentsdb and grafana, the string is a comma-separated list
-          resolutionUrl = $scope.dm_endpoints[source].split(',')[0];
-        } else if ((source === "hdfs01" || source === "HDFS") && showDefault !== true) {
+        } else if (source === "opentsdb") {
+          resolutionUrl =ConfigService.userInterfaceIndex[opentsdbIndex].split(",")[0];
```
Comment (jeclarke) [inline console-frontend/js/controllers/MetricsListCtrl.js]: space after = needed

**Review by jeclarke — CHANGES_REQUESTED** [review]
A couple of minor issues to fix up, I will go ahead and test this now anyway.

**Comment by jeclarke** [comment]
The testing was all good, it's working great. So just the couple of minor changes and I can merge this one.

**Comment by Deepa-Martin-ML** [comment]
Hi, comma is already there in Topic.json for opentsdb in between "data" and "without": ["OpenTSDB is a scalable time series database that lets you store and serve massive amounts of time series **data,without** losing granularity."]

I added the space in MetricListCtrl.js. Please let me know if anything else is required.

**Comment by jeclarke** [comment]
Sorry I meant a space needed after the comma.

**Comment by Deepa-Martin-ML** [comment]
I added the space after the comma. Please let me know if anything else is required.

**Review by jeclarke — APPROVED** [review]

**Pull request merged**